packages/headless/src/controllers/dictionary-field-context/headless-dictionary-field-context.ts

```diff
@@ -0,0 +1,98 @@
+import {CoreEngine} from '../../app/engine';
+import {dictionaryFieldContext} from '../../app/reducers';
+import {
+  addContext,
+  removeContext,
+  setContext,
+} from '../../features/dictionary-field-context/dictionary-field-context-actions';
+import {DictionaryFieldContextPayload} from '../../features/dictionary-field-context/dictionary-field-context-state';
+import {DictionaryFieldContextSection} from '../../state/state-sections';
+import {loadReducerError} from '../../utils/errors';
+import {Controller, buildController} from '../controller/headless-controller';
+
+export {DictionaryFieldContextPayload};
+
+/**
+ * The `DictionaryFieldContext` controller allows specifying which [dictionary field](https://docs.coveo.com/en/2036/index-content/about-fields#dictionary-fields) keys to retrieve.
+ */
+export interface DictionaryFieldContext extends Controller {
+  /**
+   * Sets the dictionary field context for the query. This replaces any existing context with the new one.
+   *
+   *  @param ctx - The dictionary field context to set for the query.
+   */
+  set(ctx: DictionaryFieldContextPayload): void;
```

Comment:
dont think we are gaining much from that shortening of context to ctx, I would just leave the full word

Reply:
adjusted, thanks